.github/workflows/playwright-update.yml

```diff
@@ -5,13 +5,14 @@ on:
     types: [created, edited]
 
 permissions:
-  contents: write
-  pull-requests: write
+  contents: read
```

Comment:
Wondering if this change could be the reason for the snapshot update workflow to not work anymore:

https://github.com/jupyter/notebook/actions/runs/6492244850/job/17630837598

![image](https://github.com/jupyter/notebook/assets/591645/5e680484-126a-4a91-957c-77defe4470b5)

Reply:
Hey! Yep, that seems like it. Sorry about that :/
It was difficult to interpret the needed permissions from the internal jupyterlab workflows, I only could [find that it required](https://github.com/jupyterlab/maintainer-tools/blob/main/.github/actions/update-snapshots/action.yml#L11) `pull-requests: write`
Saw that you already updated this to concede `contents: write` permission as well, that should work =)